### PR TITLE
employers page gets phone number

### PR DESCRIPTION
**Pull request by isoworg**
Please can we split test this with the existing page @pitchinvasion ?

### CODE REVIEW

File: source/employers.html.haml [hunk 1]
```diff
@@ -1,36 +1,24 @@
...
+%section.hero-splash{style: "background-image: url(#{image_path("backgrounds/pointingscreen.jpg")});"}
   .container
     .centered-row
       %h4.leader Hire brilliant
       %h1 junior developers
-      %p Sick of hearing about the talent shortage?
-      .button-row.small-padding-top
-        = link_to "Start Hiring", "/employers/contact.html", class: "button button--horizontal"
+      %p Sick of hearing about the talent shortage? <br/> Call us on:
+      %h2.leader 020 3817 8872
```
Comment (nikeshashar) [inline source/employers.html.haml]: Can this be a link? <a href="tel:+442038178872>+442038178872</a> that allows people to call straight from mobile?

**Comment by pitchinvasion** [comment]
:shipit: when the time is right for your stats

**Comment by isoworg** [comment]
Please can you also add the link that Nikesh put above? Apparently iPhones don't automatically link phone numbers, unlike Android.

We could switch today, we've only missed 8 hours of the week.

**Comment by isoworg** [comment]
`<a href="tel:+442038178872>020 3817 8872</a>`

**Comment by nikeshashar** [comment]
Thinking in a better frame of mind - let's deploy and then test on mobile. I'm probably bentincking right now

**Comment by isoworg** [comment]
Agreed.

**Comment by pitchinvasion** [comment]
@isoworg we missed this, want to try it again? when is the time right for you stats?

File: source/employers.html.haml [hunk 2]
```diff
@@ -1,36 +1,25 @@
...
+%section.hero-splash{style: "background-image: url(#{image_path("backgrounds/pointingscreen.jpg")});"}
   .container
     .centered-row
       %h4.leader Hire brilliant
       %h1 junior developers
       %p Sick of hearing about the talent shortage?
-      .button-row.small-padding-top
-        = link_to "Start Hiring", "/employers/contact.html", class: "button button--horizontal"
+      %h4.leader Call us on:
+      %h2 020 3817 8872
```
Comment (sjmog) [inline source/employers.html.haml]: %a{ href: "tel:+442038178872" } 020 3817 8872

Comment (isoworg) [inline source/employers.html.haml]: 18 july






On Wed, Jul 6, 2016 2:40 PM, Sam Morgan notifications@github.com wrote:
In source/employers.html.haml :

>    .container
>      .centered-row
>        %h4.leader Hire brilliant
>        %h1 junior developers
>        %p Sick of hearing about the talent shortage?
> -      .button-row.small-padding-top
> -        = link_to "Start Hiring", "/employers/contact.html", class: "button button--horizontal"
> +      %h4.leader Call us on:
> +      %h2 020 3817 8872


%a{ href: "tel:+442038178872" } 020 3817 8872

**Comment by pitchinvasion** [comment]
@isoworg hang on it's not quite ready, I'll edit it and the give you a squirrel and then send it live. Also your comment was very cryptic, you just added a date as a comment to Sam's comment above (I'm guessing this is partly to do with replying via email)

**Comment by isoworg** [comment]
@pitchinvasion Yes, it was supposed to be a reply to your question "when is the right time?"

Please can we go live on Monday 25 July?

**Comment by pitchinvasion** [comment]
@isoworg :shipit:

**Comment by isoworg** [comment]
@pitchinvasion :shipit: means I should merge? If so, when?

**Comment by pitchinvasion** [comment]
Whenever you're ready https://github.com/makersacademy/Engineering/blob/master/CONTRIBUTING.md

**Comment by pitchinvasion** [comment]
Welcome to the joys of continuous deployment (this is what a lot of our hiring partners struggle to do)

**Pull request merged**